### PR TITLE
[I18N] sale_validity: fr_FR translation

### PR DESCRIPTION
**Pull request by bplancher**
Hello !
I just used sale_validity module in production, which works great, i proposse now my fr_FR translation.

It's my first open source contribution, if you have advises or comments, please leave a comment ^^

### CODE REVIEW

File: sale_validity/model/sale_order.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ class sale_order(orm.Model):
             readonly=True,
             states={
                 'draft': [('readonly', False)],
-                'sent': [('readonly', True)],
+                'sent': [('readonly', False)],
```
Comment (pedrobaeza) [inline sale_validity/model/sale_order.py]: Why this modification?

Comment (bplancher) [inline sale_validity/model/sale_order.py]: Oups sorry, this commit was for our internal Odoo only.

I'll remove this commit from the pull-request ASAP.

EDIT: i removed this commit

File: sale_validity/i18n/fr.po [hunk 2]
```diff
@@ -49,6 +49,9 @@ msgid ""
 "order plus the number of days defined in this field. If the value of this "
 "field is 0, the sale orders will not have a validity date by default."
 msgstr ""
+"Par défaut, la date de validité des devis sera égale à leur date de "
+"création plus le nombre de jours saisies dans ce champs. Si ce champs "
```
Comment (yvaucher) [inline sale_validity/i18n/fr.po]: saisies sans 'es'. le nombre [...] saisi

Comment (bplancher) [inline sale_validity/i18n/fr.po]: En effet, c'est corrigé, merci !

**Comment by bplancher** [comment]
Hello, i fixed my pull-request, according to your comments.

**Comment by yvaucher** [comment]
Thanks :+1:

File: sale_validity/i18n/fr.po [hunk 3]
```diff
@@ -49,6 +49,9 @@ msgid ""
 "order plus the number of days defined in this field. If the value of this "
 "field is 0, the sale orders will not have a validity date by default."
 msgstr ""
+"Par défaut, la date de validité des devis sera égale à leur date de "
+"création plus le nombre de jours saisi dans ce champs. Si ce champs "
```
Comment (gurneyalex) [inline sale_validity/i18n/fr.po:53]: s/champs/champ/

File: sale_validity/i18n/fr.po [hunk 4]
```diff
@@ -61,3 +64,5 @@ msgid ""
 "The value of the field 'Default Validity Duration of Sale Orders' must be "
 "positive or 0."
 msgstr ""
+"La valeur du champs 'Validité par défaut des devis' doit-être supérieure "
```
Comment (gurneyalex) [inline sale_validity/i18n/fr.po:67]: s/champs/champ/

File: sale_validity/i18n/fr.po (hunk 3 above)
Comment (lepistone) [inline sale_validity/i18n/fr.po:53]: saisis? #notmynativelanguage

Comment (lepistone) [inline sale_validity/i18n/fr.po:53]: oups

**Comment by gurneyalex** [comment]
merged in 66e782b

**Pull request closed without merging**